Code/GraphMol/Wrap/Mol.cpp

```diff
@@ -199,6 +200,101 @@ namespace RDKit {
     return mol.getNumAtoms(onlyExplicit);
   }
 
+  //----------------------------------------------------------------------------
+  // Atom Bookmark Interface
+  void SetAtomBookmark(ROMol *mol, Atom *atom, int mark)
+  {
+    // dangerous as there is not concept of shared pointers here (or internal
+    //  to the underlying function.  I.e. the atom can be deleted
+    //  EVEN if stored as a bookmark
+    mol->setAtomBookmark(atom, mark);
+  }
+
+  void ReplaceAtomBookmark(ROMol *mol, Atom *atom, int mark)
+  {
+    // dangerous as there is not concept of shared pointers here (or internal
+    //  to the underlying function.  I.e. the atom can be deleted
+    //  EVEN if stored as a bookmark
+    mol->replaceAtomBookmark(atom, mark);
+  }
+
+  typedef std::vector<Atom*> AtomVector;
+  AtomVector *GetAllAtomsWithBookmark(ROMol *mol, int mark)
```

Comment:
Since this is in the Python API, why not just return a python::list (or, better, python::tuple) directly?